README.md

```diff
@@ -1,4 +1,6 @@
-# WIP Rust BLE stack
+# rubble
+
+> A Bluetooth Low-Energy stack for embedded Rust
```

Comment:
I avoided the spelled-out wording due to the copyright on the word "Bluetooth", can you change that back? Also, this isn't a quote, so the `>` should also be removed.